src/GeneralStateTestsFiller/stPreCompiledContracts2/CALLBlake2fFiller.json

```diff
@@ -0,0 +1,418 @@
+{
+  "CALLBlake2f": {
+    "env": {
+      "currentCoinbase": "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+      "currentDifficulty": "0x20000",
+      "currentGasLimit": "10000000000",
+      "currentNumber": "1",
+      "currentTimestamp": "1000",
+      "previousHash": "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+    },
+    "expect": [
+      {
+        "indexes": {
+          "data": [0],
```

Comment:
also a comment about what is the case on data 0

Reply:
described the outputs in the "// data" section, they are all hash values so they are quite opaque.